.github/workflows/enforce-labels.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   require-label:
     runs-on: ubuntu-latest
     steps:
-      - uses: mheap/github-action-required-labels@v4
+      - uses: mheap/github-action-required-labels@v5
```

Comment:
Something is amiss. This file should not have been touched.  I fear you started with the wrong baseline.